### ISSUE TITLE
Creating aliases for pipelines

### ISSUE THREAD

**Issue by thisch** [xonsh/xonsh#3891]
Creating an alias that contains a pipeline doesn't work if the alias is created using

```py
aliases['ll'] = "ls -al | more"
```

When `ll` is called, xonsh outputs

```
ls: cannot access '|': No such file or directory
ls: cannot access 'more': No such file or directory
```

How is it possible to define an alias for a command pipeline?

**Comment by scopatz** [comment]
Thanks for reporting @thisch.  I expected this too work.  But a couple of things: 

* It should probably be `aliases` in the above.
* I found that using an explicit subproc worked, i.e. `aliases['ll'] = "![ls -al | more]"`

**Comment by thisch** [comment]
Thx for the fast reply @scopatz 

> It should probably be aliases in the above.

Yes you are right, I made a typo in the description but not when testing it in my xonsh.

> I found that using an explicit subproc worked, i.e. aliases['ll'] = "![ls -al | more]"

Awesome! Thx this works.

It would be nice if this were documented in the `aliases` docu and maybe also in the bash to xonsh translation guide.

**Comment by thisch** [comment]
Your suggestion to use an explicit subproc also fixed the issue that I had with an alias that starts a program in the background:

`aliases["gk"] = "gitk --all &"` starts gitk in the foreground (-> `&` seems to be ignored) and when an explicit subproc is used it works as expected `aliases["gk"] = "![gitk --all &]"`

`aliases["gk" =

**Comment by anki-code** [comment]
It will be great to see this examples to the tutorial. PRs are welcome!

**Comment by scopatz** [comment]
Much agreed! A PR into the docs would be awesome!

**Comment by gforsyth** [comment]
I think that this usage is covered by a PR that we (certainly I) missed. 
#3677

**Comment by scopatz** [comment]
Great eye @gforsyth! I don't know how I missed that too...

**Comment by laloch** [comment]
Closed in #3677.
